src/SesWebhookCall.php

```diff
@@ -21,10 +21,10 @@ public static function storeWebhook(WebhookConfig $config, Request $request): We
         ]);
     }
 
-    public function identicalMessageExists(): bool
+    public function isFirstOfThisSesMessage(): bool
     {
-        return SesWebhookCall::where('id', '<>', $this->id)
-                                ->where('payload->MessageId', $this->payload['MessageId'])
-                                ->exists();
+        $first = SesWebhookCall::where('payload->MessageId', $this->payload['MessageId'])
```

Comment:
@freekmurze this aggregate query returns an integer properly in the app environment. When the test suite runs though, sqlite returns a string. This causes the return of this method to always be FALSE because of the different types.

Is there a workaround for this in phpunit? or do I need to change the return statement to == to allow for the different type during testing (seems like a bad idea)?

Reply:
I got this working by putting the min() query in a static method on SesWebhookCall with a return type of int.

that forces the string return from sqlite to int without having to add an explicit call to intval()

is this the proper approach?